code/modules/uplink/uplink_items.dm

```diff
@@ -1907,6 +1907,13 @@ datum/uplink_item/stealthy_tools/taeclowndo_shoes
 	restricted_roles = list("Medical Doctor", "Chief Medical Officer", "Roboticist", "Paramedic")
 	cost = 3
 
+/datum/uplink_item/rolerestricted/Syndicate_MMI
+	name = "Syndicate MMI"
+	desc = "An MMI which autmatically applies the Syndimov laws to any borg it is placed in. Great for adding known allies to assist you with a little more stealth than a fully emagged borg."
+	item = /obj/item/mmi/syndie
+	restricted_roles = list("Roboticist", "Research Director")
+	cost = 1
```

Comment:
1 is too low for effectively emag effect. I would suggest 2.

Reply:
Idk tbh. It's only laws. In addition the kit which includes tools plus restraints is 2 TC.

I could alternatively push the standalone to 2 and kit to 3

Reply:
Resolved by: 34eea88
Updating Changelog